src/javascript/app_2/App/Components/Elements/tabs.jsx

```diff
@@ -0,0 +1,141 @@
+import classNames from 'classnames';
+import React      from 'react';
+import PropTypes  from 'prop-types';
+
+class Tabs extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = { active_tab_index: '1' };
+    }
+    setActiveTab(index) {
+        this.setState({active_tab_index: index});
+    };
```

Comment:
here as well, we could remove constructor, by putting `state` outside and changing setActiveTab to arrow function